config/portals.yml

```diff
@@ -50,19 +65,19 @@ portals: &portals
     scripts:
       - edsc-portal.ornl.min.js
 
-  cwic:
-    org: CWIC
+  podaac:
+    org: PO.DAAC
     title: Search
     params:
-      tag-key:
-        - edsc.portal.cwic.collections
+      data_center:
+        - PODAAC
     logos:
-      - id: ceos-logo
-        image: /images/portals/cwic/ceos-logo.png
-        link: http://ceos.org/
-        title: CEOS Home
+      - id: podaac-logo
+        image: /images/portals/podaac/podaac-logo-mono.png
```

Comment:
This is the wrong filename for the logo you added. `podaac-logo` vs `podaac_logo`